plugins/service/renderer/srv6/srv6_renderer_test.go

```diff
@@ -79,7 +81,70 @@ type data struct {
 func TestBasicService(t *testing.T) {
 	RegisterTestingT(t)
 	retriever := configRetriever.NewMockConfigRetriever()
-	data := initTest("TestBasicService", defaultConfig(), retriever, false)
+	data := initTest("TestBasicService", defaultConfig(false), retriever, false)
+
+	// setup service
+	emptyResync(data)
+	service1 := addServiceMetadata(data, defaultPort)
+	pod1IP := addLocalPod(renderer_testing.Pod1, data, retriever)
```

Comment:
This is confusing (apart from almost no change between this and ipv4 version of test that confused github compare functionality->commenting in ipv6 test instead of ipv4 test). The only difference is boolean parameter in defaultConfig. How is this related to IPv6 vs IPv4pods/IPv6 test? If you inspect initTest and then addLocalPod, it becomes clear. Changed config in data variable change pod IP address generation in addLocalPod and you have IPv4 IP address instead of IPv6 address. Could you make it more obvious by using pod1IPv4/pod2Ipv4 instead of pod1IP/pod2IP and add comment to this line (in ipv4 test) that IP generation depends on config that is IPv4-pod oriented for ipv4 test?

Reply:
Rename